environments/prod/prod.tfvars

```diff
@@ -1607,6 +1607,34 @@ frontends = [
         ]
       },
     ],
+    disabled_rules = {
+      SQLI = [
+        "942100",
+        "942120",
+        "942150",
+        "942200",
+        "942210",
+        "942230",
+        "942370",
+        "942380",
+        "942430",
+        "942440",
+        "942450",
+      ]
+      LFI = [
+        "930100", // false positive on multi-part uploads
+        "930110", // false positive on multi-part uploads
```

Comment:
this is not the proper fix from what I remember, 
this should be added instead: https://github.com/hmcts/azure-platform-terraform/blob/DTSPO-2547-fix-whitelist-ip/environments/prod/prod.tfvars#L285-L289